doc/en/user/source/community/jdbcstore/installing.rst

```diff
@@ -0,0 +1,19 @@
+.. _community_jdbcconfig_installing:
```

Comment:
@NielsCharlier this should be ```_community_jdbcstore_installing```; the current label duplicates one used in ```jdbcconfig``` and causes a WARNING:
```/path/to/doc/en/user/source/community/jdbcstore/installing.rst:4: WARNING: duplicate label community_jdbcconfig_installing, other instance in /path/to/doc/en/user/source/community/jdbcconfig/installing.rst
```